### PR TITLE
fix(usermenu): set fixed width of user menu

### PR DESCRIPTION
**Pull request by mindreeper2420**
set fixed width of the user menu and dropdown

### CODE REVIEW

File: src/app/header/header.component.ts [hunk 1]
```diff
@@ -14,12 +14,6 @@ import { Navigation } from './../models/navigation';
 import { MenuItem } from './../models/menu-item';
 import { DummyService } from './../shared/dummy.service';
 
-
-
-interface MenuHiddenCallback {
```
Comment (joshuawilson) [inline src/app/header/header.component.ts]: You can't remove this, it breaks the build. Or else you need to find all the places that call it and change them.

Comment (mindreeper2420) [inline src/app/header/header.component.ts]: I don't recall doing anything with that, so I'll check the log again.

Comment (mindreeper2420) [inline src/app/header/header.component.ts]: @joshuawilson pushed changes and ran tests - all passed.

**Comment by fabric8cd** [comment]
@mindreeper2420 snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-950-fabric8-ui.int.rdu2c.fabric8.io

**Pull request merged**